woocommerce/woocommerce
# Add Klaviyo into default free extensions for onboarding task list

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

In this PR, we add Klaviyo into the default free extensions for onboarding marketing task list. This is shown when users turn off the marketing place suggestions in WooCommerce settings. Klaviyo should show up between Mailchimp and Creative Mail. 

Before Klaviyo is installed, it should appear in the "Recommended marketing extensions" card:

![image](https://github.com/woocommerce/woocommerce/assets/417342/b92bd096-2f0d-46f9-92fe-68c4f1490108)

After Klaviyo is installed, it should appear in the "Installed marketing extensions" card:

![image](https://github.com/woocommerce/woocommerce/assets/417342/a5469255-855d-4654-8c79-bbaed8c65952)



### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure that marketplace suggestions is **turned off** in WooCommerce settings page: `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=woocommerce_com`
2. Go to WooCommerce onboarding marketing task: `/wp-admin/admin.php?page=wc-admin&task=marketing`
3. Klaviyo should show up under the "Recommended marketing extensions" card, between Mailchimp and Creative Mail. 
4. Click on the "Get started" button for Klaviyo. This should install Klaviyo. 
5. When the installation is complete, Klaviyo should disappear from the "Recommended marketing extensions" card, and appears in the "Installed marketing extensions" card.
6. Click on the "Manage" button for Klaviyo. You should be directed to `/wp-admin/admin.php?page=klaviyo_settings` page.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
